apps/web/src/composables/users.ts

```diff
@@ -1,14 +1,26 @@
+import { ethers } from 'ethers'
+import useEnvironment from '@/composables/environment'
+import useSSV from '@/composables/ssv'
+import useWallet from '@/composables/wallet'
 import { ProviderString } from '@/types/ProviderString'
 import { User } from '@/interfaces/User'
-import { ref } from 'vue'
+import { onMounted, ref } from 'vue'
+
+const user = ref<User>({
+    id: '0xd557a5745d4560B24D36A68b52351ffF9c86A212',
+    accounts: {
```

Comment:
Is this the only information we can get on a user?
Is it possible to get the account name or hardcode it somewhere?

Reply:
Sounds good, I noted your use of the account name and hardcoded it.